### ISSUE TITLE
Confirm proper usage

### ISSUE THREAD

**Issue by JadBatmobile** [rishizek/tensorflow-deeplab-v3#31]
Hello,

I would like to confirm i am properly using the tensorflow-deeplab-v3 repo

I have my own 3 class semantic segmentation data set. The images are RGB, with black, red, and green denoting the classes.

I used the masks_kitty.pynb script referenced to convert my data to grayscale, with pixel values denoting class index (0, 1 and 2).

I then used the create_pascal_tf_record.py script provided to create tensorflow records.

My data does not resemble the data mentioned in SegmentationClassAug, which still confuses me, since i do not know what the intention is with data where object outlines are shown only. Unlike the VOC dataset, my segmentation dataset does not have a white outline around the objects. It only has the objects colored accordingly, so im not sure what the connection between SegmentationClassAug and other datasets is.

I used the following command to train:
python train.py --pre_trained_model /media/hallab/333B50E8464A610F/resnet_v2_101_2017_04_14/resnet_v2_101.ckpt --batch_size 1 --data_dir ~/deep_data/ 

where ~/deep_data contains two files "voc_train.record" and "voc_val.record", generated by create_pascal_tf_record.py 

NOTE: i was not including --model_dir specifier because the pretrained model did not seem compatible with the resnet_v2_101

**Comment by eleflea** [comment]
The white outline in VOC dataset is denotes 255 in value, which means you should ignore those pixels. I think it's okay to use a dataset without outline. You can use create_pascal_tf_record.py, but need to perpare train.txt and val.txt yourself. And your label image should be grayscale png(0, 1, 2).
The --model_dir specifier is where to save your model.

**Comment by JadBatmobile** [comment]
okay!
